src/tests/soft_asserts.rs

```diff
@@ -20,6 +22,7 @@ pub fn soft_assert_eq2<T: Debug + PartialEq + Eq, H: FnOnce() -> String>(v1: T,
     }
 }
 
+/// Inlined test of whether [vectors](Vector) `v1 == v2`.
```

Comment:
Might want to add here something like "While it is equivalent to soft_assert_eq2, this version will print a more readable error message on failure"